packages/terra-popup/src/_PopupContent.jsx

```diff
@@ -250,8 +250,11 @@ class PopupContent extends React.Component {
 
 PopupContent.propTypes = propTypes;
 PopupContent.defaultProps = defaultProps;
-PopupContent.Opts = {
+
+const PopupContentWithIntl = injectIntl(PopupContent);
+
+PopupContentWithIntl.Opts = {
   cornerSize: CORNER_SIZE,
 };
 
-export default injectIntl(PopupContent);
+export default PopupContentWithIntl;
```

Comment:
This might be better solved exporting the constant separately.
```
export default injectIntl(PopupContent);
export {
  CORNER_SIZE as cornerSize,
};
```

Reply:
I'm okay with either approach. I mentioned something similar to what you describe in #513

![image](https://user-images.githubusercontent.com/34115417/52820897-df2a3100-3072-11e9-9ea7-381c0550d434.png)

The current method just involves less moving parts and is passive.

Reply:
Let's stick with the passive solution.

Reply:
It would be passive in either case. Corner size is only exposed internally for the use of Popup.jsx itself here:
https://github.com/cerner/terra-framework/blob/d0360e660cb4ce8e99c73a2ba366971825015689/packages/terra-popup/src/Popup.jsx#L141

I lean toward exposing the value by itself, potentially updating arrow to match the same pattern.

Reply:
Sounds good to me, let's move forward with that @dev-cprice